charts/cluster-addons/templates/monitoring/loki-stack.yaml

```diff
@@ -102,31 +117,44 @@ spec:
           release: loki-stack
         action: delete
     {{- end }}
-    # After deleting the release, remove the PVCs
-    postDelete:
-      - apiVersion: v1
-        kind: PersistentVolumeClaim
-        selector:
-          app: loki
-          release: loki-stack
-        action: delete
+
+---
+apiVersion: addons.stackhpc.com/v1alpha1
+kind: HelmRelease
+metadata:
+  name: {{ include "cluster-addons.componentName" (list . "promtail") }}
+  labels: {{ include "cluster-addons.componentLabels" (list . "promtail") | nindent 4 }}
+spec:
+  clusterName: {{ include "cluster-addons.clusterName" . }}
+  bootstrap: true
+  chart: {{ toYaml .Values.monitoring.promtail.chart | nindent 4 }}
+  targetNamespace: {{ .Values.monitoring.promtail.release.namespace }}
+  releaseName: promtail
+  valuesSources:
+    - secret:
+        name: {{ include "cluster-addons.componentName" (list . "promtail") }}-config
+        key: defaults
+    - secret:
+        name: {{ include "cluster-addons.componentName" (list . "promtail") }}-config
+        key: overrides
+{{- end }}
```

Comment:
If this `{{- end }}` is for the `if` clause in line 1 of the file then I think should be add the bottom of the file? At the moment I think the loki-dashboards Manifest is included even if Loki is disabled which is not what we want.